db/schema.prisma

```diff
@@ -8,26 +8,49 @@ datasource db {
   relationMode = "prisma" // to not worry about foreign ket constrains 
 }
 
+
 model User {
```

Comment:
We dont need a user table that tracks username and gives the user a new id. It creates redundency and leaves room for errors (if somehow clerk has a different username than us then we have a problem)